### PR TITLE
Update GitHub links & Make markdown extension consistent

### PR DESCRIPTION
**Pull request by jamen**
I couldn't remember what the images in the readme were before they broke, so those were my best guess.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -736,7 +736,7 @@ your package.json dependencies field.
 ## b.on('file', function (file, id, parent) {})
 ## b.pipeline.on('file', function (file, id, parent) {})
 
-When a file is resolved for the bundle, the bundle emits a `'file'` event with
+When a file is browserifyresolved for the bundle, the bundle emits a `'file'` event with
```
Comment (terinjokes) [inline readme.md]: looks like a mistake here

File: readme.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ browserify will recursively analyze all the `require()` calls in your app in
 order to build a bundle you can serve up to the browser in a single `<script>`
 tag.
 
-[![build status](https://img.shields.io/travis/substack/node-browserify/master.svg)](https://travis-ci.org/substack/node-browserify)
+[![build status](https://img.shields.io/travis/browserify/browserify/master.svg)](https://travis-ci.org/substack/node-browserify)
```
Comment (terinjokes) [inline readme.md]: You should update the link component as well.

**Comment by jamen** [comment]
I just noticed, every repo uses `.markdown`.  So I could do `code-of-conduct.markdown` instead of switching to `.md`.

**Comment by terinjokes** [comment]
My preference is "md", but I don't want to stop this effort with a thousand cuts. Pick whatever looks the most consistent and use that :smile:

**Comment by abouthiroppy** [comment]
@jamen please need to rebase;)

**Comment by jamen** [comment]
Solved by 0305b703b226878f3acb5b8f2ff9451c87cd3991 :)

**Pull request closed without merging**